### PR TITLE
Add pinout cli tool, close #444, close #497

### LINKED ISSUE #444: pinout command line tool
We currently have a `pi_info` function which allows you to find out information on the Pi you're on, or another specified Pi. We want to add a command line tool `pinout` which would print the pinout for your Pi by default, or a specified Pi.

See #354 for info on how to get pinout from pi_info.

An example of argparse: https://github.com/pyjokes/pyjokes/blob/master/scripts/pyjoke

### PR DESCRIPTION
**Pull request by bennuttall**
New PR introducing small changes to @stewartadcock's pinout cli tool PR #497.

@waveform80 I'm about to make a few notes in a code review but can you please check it over? Looks good to me, it works, and tests pass!

### CODE REVIEW

File: docs/cli_tools.rst [hunk 1]
```diff
@@ -0,0 +1,26 @@
+==================
+Command-line Tools
+==================
+
+Pinout
+======
+
+The gpiozero package contains a database of information about the various
+revisions of Raspberry Pi. This is queried by the ``pinout`` command-line
+tool to write details of the GPIO pins available.
```
Comment (lurch) [inline docs/cli_tools.rst]: Perhaps this should use "display" instead of "write"?

**Comment by lurch** [comment]
@bennuttall One of the other things we discussed "fixing" was changing the `exit` calls to use sys.exit, and changing all errors to consistently return an error-code of `1`.
(unless @stewartadcock can explain why he's used `exit(-1)` and `exit(2)` ??)

File: docs/cli_tools.rst (hunk 1 above)
Comment (stewartadcock) [inline docs/cli_tools.rst]: My thought is that "display" could be misleading. How about "output"?

Depends on the audience. Thinking of my own children, for end of KS2 "output" seems a good choice, but earlier I'd stick to "write"/"display" or, maybe, "give".

Comment (lurch) [inline docs/cli_tools.rst]: Yeah, "output" matches the wording used in `pinout.py` itself. Fixed.

**Comment by lurch** [comment]
> I suggest the exit codes are changed as suggested.

Done :)

**Comment by waveform80** [comment]
Okay, been over most of this today and it looks good to me - sorry for the long delay getting back to GPIO Zero, but it looks like everyone's been busy in my absence! Will try to get on with plenty of bits this week :)

**Comment by lurch** [comment]
Nice to "see" you again @waveform80 :smiley:

**Pull request merged**